TopAnalysis/interface/FCNCTriLeptonCppWorker.h

```diff
@@ -32,6 +32,16 @@ class FCNCTriLeptonCppWorker {
   void resetValues();
   bool analyze();
 
+  float get_LeadingMuon_pt() const { return out_LeadingMuon_p4[0]; }
+  float get_LeadingMuon_eta() const { return out_LeadingMuon_p4[1]; }
+  float get_LeadingMuon_phi() const { return out_LeadingMuon_p4[2]; }
+  float get_LeadingMuon_mass() const { return out_LeadingMuon_p4[3]; }
+
+  float get_LeadingElectron_pt() const { return out_LeadingElectron_p4[0]; }
+  float get_LeadingElectron_eta() const { return out_LeadingElectron_p4[1]; }
+  float get_LeadingElectron_phi() const { return out_LeadingElectron_p4[2]; }
+  float get_LeadingElectron_mass() const { return out_LeadingElectron_p4[3]; }
+
   float get_LeadingLepton_pt() const { return std::max(std::max(get_Lepton1_pt(), get_Lepton2_pt()), get_Lepton3_pt()); }
```

Comment:
change this to std::max(out_LeadingMuon_p4[0], out_LeadingElectron_p4[0]);

Reply:
Should we see the leading muon/electron regardless of channel?
In NTU presentation, they drew the leading muon when the 3lepton has >= 2mu and leading electron when the 3 lepton has >=2el.
Or do you mean return should be changed like "std::max(out_Lepton2_p4[0], out_Lepton3_p4[0])" at the line 41 to 43..?